bpf/lib/overloadable_skb.h

```diff
@@ -25,7 +26,11 @@ bpf_clear_meta(struct __sk_buff *ctx)
 static __always_inline __maybe_unused int
 get_identity(const struct __sk_buff *ctx)
```

Comment:
Since these are changing now, it maybe cool to try to implement a BPF unit tests for the functions you change, assuming there aren't limitations to doing so.

Reply:
Good suggestion! I added some unit tests in the most recent commit